include/core/utilities.h

```diff
@@ -231,20 +231,25 @@ create_output_folder(const std::string &dirname);
  * -------
  * Tracer
  * -------
- * * @param expression string that will be printed
+ *
+ * @param pcout the parallel cout used to print the information
+ * @param expression string that will be printed
+ * @param delimiter the character used to delimit the printing. Default value is "-"
  */
 inline void
-announce_string(const ConditionalOStream &pcout, const std::string expression)
+announce_string(const ConditionalOStream &pcout,
+                const std::string         expression,
+                const char                del = '-')
 {
   for (unsigned int c = 0; c < expression.size() + 1; ++c)
```

Comment:
There is no need to loop here. You can use the string constructor and do `pcout << std::string(expression.size(), del)`

Reply:
And I would use `output` and `delimiter`